apps/minifront/src/components/ibc/ibc-in/cosmos-wallet-connector.tsx

```diff
@@ -13,7 +13,7 @@ export const CosmosWalletConnector = () => {
 
   return (
     <div className='flex flex-col items-center justify-center gap-4'>
-      <div className='w-52'>
+      <div className='flex w-52 flex-col justify-center'>
```

Comment:
Removing `interchain-ui` button usage requires parent to define sizing of children